mon-entreprise/scripts/search/update-data.js

```diff
@@ -0,0 +1,162 @@
+require('dotenv').config()
+
+const algoliasearch = require('algoliasearch')
+const rules = require('modele-social')
+const getSimulationData = require('../../source/pages/Simulateurs/metadata-src')
+
+const {
+	ALGOLIA_APP_ID,
+	ALGOLIA_ADMIN_KEY,
+	ALGOLIA_INDEX_PREFIX = '',
+} = process.env
+
+console.log('ALGOLIA APP ID: ' + ALGOLIA_APP_ID)
+
+const client = algoliasearch(ALGOLIA_APP_ID, ALGOLIA_ADMIN_KEY)
+
+const rulesIndex = client.initIndex(`${ALGOLIA_INDEX_PREFIX}rules`)
+const simulateursIndex = client.initIndex(`${ALGOLIA_INDEX_PREFIX}simulateurs`)
+
+const formatRulesToAlgolia = (rules) => {
+	const ruleEntries = Object.entries(rules)
+	return ruleEntries
+		.map(([n, rule]) => {
+			if (rule) {
```

Comment:
Personnellement, je trouve que c'est une meilleure pratique d'avoir le cas de retour exception explicité en début de code (voir par exemple cette discussion : https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement)

Reply:
Ça vaudrait le coup de l'avoir sous forme de regle ESlint